src/dali_executor/CMakeLists.txt

```diff
@@ -48,6 +48,8 @@ add_custom_command(  # Download and unpack DALI wheel
     COMMAND
         # I've spent a whole day writing the magical line below.
         /bin/bash -c "for file in $(ls ${CMAKE_CURRENT_BINARY_DIR}/*.whl); do unzip -q -o $file -d ${CMAKE_CURRENT_BINARY_DIR}/dali; done"
+    COMMAND
+        /bin/bash -c "for file in $(ls ${CMAKE_CURRENT_BINARY_DIR}/*.tar.gz); do tar -xzf $file -C ${CMAKE_CURRENT_BINARY_DIR}/dali; done"
```

Comment:
It assumes that the sdiist doesn't execute any installation logic. What if it compiles a source file?

Reply:
just wondering: why not doing pip install?

Reply:
That's a very good point. I'll try to figure something else out. Possibly I can `pip install --target` and adjust `PYTHONPATH` afterwards.

Reply:
Now it should work. `PYTHONPATH` needs to be adjusted anyway I guess.